Add testing docs generated by test-metadata-generator plugin (#101)
skodjob/odh-e2e: docs/md/io/odh/test/e2e/standard/PipelineServerST.md

| Step | Action | Result |
| - | - | - |
| 1. | Deploy Pipelines Operator | Pipelines operator is available on the cluster |
| 2. | Deploy ServiceMesh Operator | ServiceMesh operator is available on the cluster |
| 3. | Deploy Serverless Operator | Serverless operator is available on the cluster |
| 4. | Install ODH operator | Operator is up and running and is able to serve it's operands |
| 5. | Deploy DSCI | DSCI is created and ready |
| 6. | Deploy DSC | DSC is created and ready |

**After tests execution steps:**

| Step | Action | Result |
| - | - | - |
| 1. | Delete ODH operator and all created resources | Operator is removed and all other resources as well |

<hr style="border:1px solid">

## testUserCanCreateRunAndDeleteADSPipelineFromDSProject

**Description:** Check that user can create, run and deleted DataSciencePipeline from a DataScience project

**Contact:** `Jiri Danek <jdanek@redhat.com>`

**Steps:**

| Step | Action | Result |
| - | - | - |
| 1. | Create namespace for DataSciencePipelines application with proper name, labels and annotations | Namespace is created |
| 2. | Create Minio secret with proper data for access s3 | Secret is created |

> | 3. | Create DataSciencePipelines application with configuration for new Minio instance and new MariaDB instance | Notebook resource is created |

Reviewer: ```suggestion
| 3. | Create DataSciencePipelinesApplication with configuration for new Minio instance and new MariaDB instance | DataSciencePipelinesApplication resource is created |
```

Reviewer: 🤦🏽 ofc I'm commenting the generated file and not the source

> | 4. | Wait for DataSciencePipelines server readiness | DSPA endpoint is available and it return proper data |

Reviewer: ```suggestion
| 4. | Wait for DataSciencePipelines API server readiness | DSP API endpoint is available |
```
